packages/components/slider/package.json

```diff
@@ -56,6 +56,7 @@
     "@yamada-ui/utils": "workspace:*"
   },
   "devDependencies": {
+    "@testing-library/user-event": "^14.5.2",
```

Comment:
@neginasu 
You don't need to install it.

Reply:
@koralle 
Thanks for the review.
I have uninstalled the package.